### PR TITLE
Simplify ROCm install guide

### PR DESCRIPTION
**Pull request by kmaehashi**

### CODE REVIEW

File: docs/source/install_rocm.rst [hunk 1]
```diff
@@ -1,66 +1,28 @@
...
 
-You need to have the following components to use CuPy.
+* `AMD GPU supported by ROCm <https://github.com/RadeonOpenCompute/ROCm#Hardware-and-Software-Support>`_
 
-* GPU supported by ROCm (AMD GPUs or NVIDIA GPUs)
-* `ROCm <https://rocm.github.io/install.html>`_
-    * Supported Versions: ROCm 2.6+.
-* `Python <https://python.org/>`_
-* `NumPy <http://www.numpy.org/>`_
+* `ROCm <https://rocmdocs.amd.com/en/latest/index.html>`_: v2.6+
```
Comment (emcastillo) [inline docs/source/install_rocm.rst:11]: @leofang is the support limited only to 3.5?

Comment (leofang) [reply docs/source/install_rocm.rst:11]: Currently, yes. I will create a PR to support 3.5+.

Comment (emcastillo) [reply docs/source/install_rocm.rst:11]: I think we should change the 2.6 here for 3.5 right?

Comment (leofang) [reply docs/source/install_rocm.rst:11]: > I will create a PR to support 3.5+.

See #4110. I think with that quick fix it should be safe to say 3.5.0+ here, although I'll need to find some time to test 3.8.0 too.

Comment (leofang) [reply docs/source/install_rocm.rst:11]: btw We definitely don't wanna mention ROCm 2.x anymore, as I think a lot of components are incompatible now.

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Review by emcastillo — APPROVED** [review]

**Pull request merged**